tests/api/test_statements_get.py

```diff
@@ -42,22 +42,30 @@ def setup_es_index(statements):
             for statement in statements
         ],
     )
-    ES_INDICES_CLIENT.refresh()
+    es_client.indices.refresh()
 
 
-@pytest.fixture(autouse=True)
-def teardown_es_index():
-    """
-    Clean up Elasticsearch after each test.
-    """
-    yield
-    ES_INDICES_CLIENT.delete(index=ES_TEST_INDEX)
+def insert_mongo_statements(mongo_client, _, statements):
+    """Inserts a bunch of example statements into MongoDB for testing."""
 
+    database = getattr(mongo_client, MONGO_TEST_DATABASE)
+    collection = getattr(database, MONGO_TEST_COLLECTION)
+    collection.insert_many(list(MongoDatabase.to_documents(statements)))
```

Comment:
Not a big fan of this trick :wink: Why not having a single `insert_statements(client, statements)` and test if the `client` is a mongo or ES client instance to switch between methods?

Reply:
Great Idea) Indeed, this isn't ideal, I tried to put the fixture as a parameter and stumbled on [an pytest issue](https://github.com/pytest-dev/pytest/issues/349)( However, it suggests a workaround using a parametrized fixture. Updated the tests with their approach) WDYT?